x-pack/plugins/snapshot_restore/public/shim.ts

```diff
@@ -0,0 +1,104 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { I18nContext } from 'ui/i18n';
+
+import chrome from 'ui/chrome';
+import { management, MANAGEMENT_BREADCRUMB } from 'ui/management';
+import { fatalError } from 'ui/notify';
+import routes from 'ui/routes';
+
+import { unmountComponentAtNode } from 'react-dom';
+import { HashRouter } from 'react-router-dom';
+
+export interface AppCore {
```

Comment:
when this PR or https://github.com/elastic/kibana/pull/32746 finished we need to update typings

Reply:
thanks for the heads up 😄 I am happy to update these typings when that PR is merged. work with continue for this feature in subsequent PRs into the main feature branch, so it will be easy to update since there won't be any conflicts with master